packages/mds-agency/request-handlers.ts

```diff
@@ -59,17 +57,33 @@ export const registerVehicle = async (req: AgencyApiRegisterVehicleRequest, res:
   const { body } = req
   const recorded = now()
 
-  const { provider_id } = res.locals
-  const { device_id, vehicle_id, type, propulsion, year, mfgr, model } = body
-
-  const status = VEHICLE_STATUSES.removed
-
+  const { provider_id, version } = res.locals
+  if (!version || version === '0.4.1') {
```

Comment:
I think it would be better to use a named constant instead of the version number directly.